packages/core/src/components/deprecated-text/HeaderText.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from "react";
+import { Heading, HeadingProps } from "../heading/Heading";
+
+/**
+ * @deprecated Please use `Heading` instead.
+ */
+export const HeaderText: React.FC<Omit<HeadingProps, "variant">> = (props) => {
+  return <Heading variant={"h2"} {...props} />;
```

Comment:
This was `<h1>` before?

![image](https://user-images.githubusercontent.com/817046/100199882-2018d000-2efe-11eb-8517-886b77509940.png)

Reply:
Yes, but all headings was h1, but with different fontSizes set.
h2 is visually closest to the old HeaderText.